tee-worker/ts-tests/identity.test.ts

```diff
@@ -389,6 +393,38 @@ describeLitentry('Test Identity', (context) => {
         });
     });
 
+    step('remove error identities', async function () {
+        //remove a nonexistent identity from an account
+        const resp_not_exist_identities = (await removeErrorIdentities(context, context.defaultSigner[0], true, [
+            twitterIdentity,
+            ethereumIdentity,
+            substrateIdentity,
+        ])) as string[];
+
+        resp_not_exist_identities.map((item: any) => {
+            const result = item.toHuman().data.reason;
+            assert(
+                result.search('IdentityNotExist') !== -1,
+                'remove twitter should fail with reason `IdentityNotExist`'
+            );
+        });
+
+        //remove a challenge code before the code is set
+        const resp_not_created_identities = (await removeErrorIdentities(context, context.defaultSigner[2], true, [
+            twitterIdentity,
+            ethereumIdentity,
+            substrateIdentity,
+        ])) as string[];
+
+        resp_not_created_identities.map((item: any) => {
+            const result = item.toHuman().data.reason;
+            assert(
+                result.search('IdentityNotExist') !== -1,
+                'remove twitter should fail with reason `IdentityNotExist`'
+            );
+        });
+    });
+
```

Comment:
1. This step seems identical with the previous one, we should have comment at least about why we repeat it
2. even though we repeat it - we should have a different step name
3. we should avoid duplicate code on step-level
4. we should avoid duplicate code inside a step: I don't see any difference between `const resp_not_exist_identities...` and `const resp_not_created_identities...` except for the signer. Can't we use a function to wrap it up?

Reply:
The difference between the two removeErrorIdentities is that there is no challengeCode for `context.defaultSigner[2]`,so for the convenience of testing, I directly change the signer and repeat it.